MediaBrowser.Controller/MediaEncoding/EncodingHelper.cs

```diff
@@ -2653,26 +2653,121 @@ public string GetAudioFilterParam(EncodingJobInfo state, EncodingOptions encodin
 
             var filters = new List<string>();
 
-            if (channels.HasValue
-                && channels.Value == 2
-                && state.AudioStream is not null
-                && state.AudioStream.Channels.HasValue
-                && state.AudioStream.Channels.Value == 6)
+            if (channels is 2 && state.AudioStream?.Channels is > 2)
             {
                 if (!encodingOptions.DownMixAudioBoost.Equals(1))
                 {
                     filters.Add("volume=" + encodingOptions.DownMixAudioBoost.ToString(CultureInfo.InvariantCulture));
                 }
 
-                switch (encodingOptions.DownMixStereoAlgorithm)
+                switch (state.AudioStream.Channels.Value)
                 {
-                    case DownMixStereoAlgorithms.Dave750:
-                        filters.Add("pan=stereo|c0=0.5*c2+0.707*c0+0.707*c4+0.5*c3|c1=0.5*c2+0.707*c1+0.707*c5+0.5*c3");
+                    case 3:
```

Comment:
Just thinking out loud here, but maybe this would be preferable to nested switches. Thoughts?
```suggestion
                    case 3 when encodingOptions.DownMixStereoAlgorithm == ownMixStereoAlgorithms.Rfc7845:
```

Reply:
I plan to add more algorithms later and will modify this in that one.